tests/pages/services/ServiceReviewScreen-cy.js

```diff
@@ -0,0 +1,1646 @@
+require("../../_support/utils/ServicesUtil");
```

Comment:
There is A LOT of repetition in this file that perhaps we could re-use some parts, not a blocker too since I expect isolated tests that way making more stable

Reply:
Maybe we can identify the differences and combine the two correlating ones in a follow-up PR 🤔  I think I would like to give that  a try to learn a bit more about our setup